x-pack/legacy/plugins/ml/public/services/ml_api_service/index.d.ts

```diff
@@ -69,6 +69,7 @@ declare interface Ml {
     getDataFrameAnalyticsStats(analyticsId?: string): Promise<GetDataFrameAnalyticsStatsResponse>;
     createDataFrameAnalytics(analyticsId: string, analyticsConfig: any): Promise<any>;
     evaluateDataFrameAnalytics(evaluateConfig: any): Promise<any>;
+    estimateMemoryUsage(jobConfig: any): Promise<any>;
```

Comment:
could you please provide types here?

Reply:
Added type: b63f93e177edf6bbfb133a3a5505aae67cde091b